server/pages/serviceWorker.js

```diff
@@ -0,0 +1,141 @@
+	var now = Date.now();
+
+	var urlsToPrefetch = [
+		'http://fonts.gstatic.com/s/opensans/v13/cJZKeOuBrn4kERxqtaUH3ZBw1xU1rKptJj_0jans920.woff2',
+		'http://fonts.gstatic.com/s/opensans/v13/xjAJXh38I15wypJXxuGMBogp9Q8gbYrhqGlRav_IXfk.woff2',
+		'http://fonts.gstatic.com/s/opensans/v13/DXI1ORHCpsQm3Vp6mXoaTRampu5_7CjHW5spxoeN3Vs.woff2',
+		'http://fonts.gstatic.com/s/merriweather/v8/RFda8w1V0eDZheqfcyQ4EBampu5_7CjHW5spxoeN3Vs.woff2',
+		'http://s1.wp.com/wp-includes/js/tinymce/skins/lightgray/content.min.css',
+		'http://s1.wp.com/wp-includes/js/tinymce/skins/lightgray/skin.min.css',
+		'https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,400,300,600|Merriweather:700,400,700italic,400italic',
+		'https://s1.wp.com/i/noticons/noticons.css?v=20150727',
+		'https://s1.wp.com/wp-includes/css/dashicons.css?v=20150727'
+	];
+
+	// All of these logging statements should be visible via the "Inspect" interface
+	// for the relevant SW accessed via chrome://serviceworker-internals
+	console.log( 'Handling install event. Resources to pre-fetch:', urlsToPrefetch );
+
+	event.waitUntil(
+		caches.open( CURRENT_CACHES.prefetch).then( function( cache ) {
+			var cachePromises = urlsToPrefetch.map( function( urlToPrefetch ) {
+				// This constructs a new URL object using the service worker's script location as the base
+				// for relative URLs.
+				var url = new URL( urlToPrefetch, location.href );
+				// Append a cache-bust=TIMESTAMP URL parameter to each URL's query string.
+				// This is particularly important when precaching resources that are later used in the
+				// fetch handler as responses directly, without consulting the network (i.e. cache-first).
+				// If we were to get back a response from the HTTP browser cache for this precaching request
+				// then that stale response would be used indefinitely, or at least until the next time
+				// the service worker script changes triggering the install flow.
+				url.search += ( url.search ? '&' : '?' ) + 'cache-bust=' + now;
+
+				// It's very important to use {mode: 'no-cors'} if there is any chance that
+				// the resources being fetched are served off of a server that doesn't support
+				// CORS (http://en.wikipedia.org/wiki/Cross-origin_resource_sharing).
+				// In this example, www.chromium.org doesn't support CORS, and the fetch()
+				// would fail if the default mode of 'cors' was used for the fetch() request.
+				// The drawback of hardcoding {mode: 'no-cors'} is that the response from all
+				// cross-origin hosts will always be opaque
+				// (https://slightlyoff.github.io/ServiceWorker/spec/service_worker/index.html#cross-origin-resources)
+				// and it is not possible to determine whether an opaque response represents a success or failure
+				// (https://github.com/whatwg/fetch/issues/14).
+				return fetch( new Request( url, { mode: 'no-cors' } ) ).then( function( response ) {
+					if ( response.status >= 400 ) {
```

Comment:
With `mode: 'no-cors' ` you will receive opaque response. The [status for these responses is always 0](https://fetch.spec.whatwg.org/#concept-filtered-response-opaque) cause it's intended you can extract no knowledge about them at all.